components/managed-identity/managed-identity.tf

```diff
@@ -77,32 +75,47 @@ locals {
 
   # MIs for managed-identities-sandbox-rg etc - for workload identity with ASO
   mi_cft = {
+    # DCD-CNP-Sandbox
     sbox = {
-      scope = "/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783"
-    }
-    perftest = {
-      scope = "/subscriptions/7a4e3bd5-ae3a-4d0c-b441-2188fee3ff1c"
+      scope  = "/subscriptions/bf308a5c-0624-4334-8ff8-8dca9fd43783"
```

Comment:
remove this scope, change it to `subscription_id` and refactor what ever was using this before with a local variable, something like:

```
subscription_scope = "/subscriptions/${local.mi_cft[var.env].subscription_id}"
```

Reply:
on it